src/controllers/authController.js

```diff
@@ -5,6 +5,36 @@ const CustomError = require('../errors');
 const { attachCookiesToResponse, createTokenUser } = require('../utils');
 
 const register = async (req, res) => {
+/* 
+    #swagger.summary = 'Register a new user'
+    #swagger.parameters['user'] = {
+      in: 'body',
+      description: 'User registration information',
+      required: true,
+      schema: {
+        type: 'object',
+        properties: {
+          name: { type: 'string' },
+          email: { type: 'string' },
+          password: { type: 'string' },
+          username: { type: 'string' }
+        }
+      }
+    }
+    #swagger.responses[201] = {
+      description: 'User registered successfully',
+      schema: {
+        $ref: '#/definitions/User' // Reference to the User object definition
+      }
+    }
+    #swagger.responses[400] = {
+      description: 'Bad request, missing email, name, password, or username'
+    }
+    #swagger.responses[409] = {
+      description: 'Conflict, email already exists'
+    }
```

Comment:
I agree that this should return a 409 but looking at the code on line 46, right now we're just returning a 400. So either have the docs match what's in the code or vice versa

Reply:
Removed 409